build-linux.sh

```diff
@@ -13,5 +13,5 @@ docker run -it --rm \
   -u jenkins:$(getent group $(whoami) | cut -d: -f3) \
   -v "${PWD}:/status-desktop" \
   -w /status-desktop \
-  statusteam/nim-status-client-build:1.0.2 \
```

Comment:
That's not gonna fly, as I don't update the `latest` flag normally. Better to keep it up to date.
And the current tag is [1.1.3](https://hub.docker.com/layers/statusteam/nim-status-client-build/1.1.3/images/sha256-e345051d26e5a1775ba78a395516f123781cacf2b32d0d692bdf42a11f60990f?context=explore).